src/Pages/3DPrinting/3DPrintForm.js

```diff
@@ -76,38 +78,48 @@ export default class PrintForm3D extends React.Component {
   checkIfFormFilled () {
     fill = true
     if (this.state.name.length === 0) {
-      window.alert('You must provide your full name!')
+      this.setState({ error: 'You must provide your full name!' })
       fill = false
     } else if (
       this.state.color.length === 0 ||
       this.state.color === 'Select Color'
     ) {
-      window.alert('You must provide a color')
-      fill = false
-    } else if (this.state.url.length === 0) {
-      window.alert('You must provide a url')
+      this.setState({ error: 'You must provide a color' })
       fill = false
     } else if (this.state.projectType.length === 0) {
-      window.alert('You must provide your type of project')
+      this.setState({ error: 'You must provide your type of project' })
       fill = false
-    } else if (this.state.contact.length === 0) {
-      window.alert('You must provide a contact')
+    } else if (this.state.url.length === 0) {
+      this.setState({ error: 'You must provide a url' })
       fill = false
     }
+    if (!fill) {
+      this.displayErrorMessage()
+    }
     return fill
   }
 
+  // Displays error message if a form input is empty
+  displayErrorMessage () {
+    document.getElementById('error-message').style.display = 'block' // Show
+    setTimeout(() => {
+      document.getElementById('error-message').style.display = 'none' // Hide
+    }, 2000)
+  }
+
   async submitApplication () {
     if (this.checkIfFormFilled()) {
+      const { name } = this.props.user
```

Comment:
```suggestion
      const { name, email } = this.props.user
```

Reply:
fixed

Reply:
Next time resolve the conversation if they are fixed